fuzz/fuzz_targets/fuzz_name.rs

```diff
@@ -0,0 +1,22 @@
+#![no_main]
+use std::error::Error;
+
+use libfuzzer_sys::fuzz_target;
+use skrifa::{raw::TableProvider, FontRef};
+
+fn do_name_things(data: &[u8]) -> Result<(), Box<dyn Error>> {
+    let font = FontRef::new(data)?;
+    let name = font.name()?;
+
+    for nr in name.name_record() {
+        let _ = nr.string(name.string_data());
```

Comment:
Suggest adding `.map(|ns| ns.to_string())` to exercise the string data parsing.

Reply:
Fixed, ty